extension/content.css

```diff
@@ -163,3 +163,16 @@
 .file-navigation a[href$="/upload"] {
 	display: none !important;
 }
+
+/* style for edit README button */
+#readme #refined-github-readme-edit-link {
+	position: absolute;
+	top: 10px;
+	right: 10px;
+	opacity: 0;
+	transition: opacity 250ms;
+}
+
+#readme:hover #refined-github-readme-edit-link {
```

Comment:
Nice. Also like the 0.2 default better. Reduced the specificity, no need for the `#readme` selector since it's already an ID.